java/fury-core/src/main/java/org/apache/fury/logging/LoggerFactory.java

```diff
@@ -47,7 +47,7 @@ public static void useSlf4jLogging(boolean useSlf4jLogging) {
 
   public static Logger getLogger(Class<?> clazz) {
     if (disableLogging) {
-      return new NilLogger();
+      return new FuryLogger(clazz, logLevel);
```

Comment:
I have a confusion, why we disableLogging, but still use `FuryLogger` to output Log.

Reply:
The logging is disabled in https://github.com/apache/incubator-fury/pull/1492 disabled logging, I changed this for debug but forget to revert it. I will fix it in another PR